commit 8a8f322728b40533a1108b798bbb6329d4fb4e86
Author: Tim Foster <tim.foster@joyent.com>
Date:   2019-05-20T16:19:57+01:00 (5 months ago)
    
    OS-7799 build_jenkins should use 'public' not 'stor' for the destination manta path
    Reviewed by: Jerry Jelinek <jerry.jelinek@joyent.com>
    Approved by: Jerry Jelinek <jerry.jelinek@joyent.com>

diff --git a/tools/build_jenkins b/tools/build_jenkins
index dc4d28be..0660618e 100755
--- a/tools/build_jenkins
+++ b/tools/build_jenkins
@@ -6,7 +6,7 @@
 #
 
 #
-# Copyright (c) 2019, Joyent, Inc.
+# Copyright 2019 Joyent, Inc.
 #
 
 #
@@ -24,7 +24,7 @@ set -o pipefail
 
 # Allow users to override the Manta path where we publish artifacts
 if [[ -z "${ENGBLD_DEST_OUT_PATH}" ]]; then
-    export ENGBLD_DEST_OUT_PATH=/stor/builds
+    export ENGBLD_DEST_OUT_PATH=/public/builds
 fi
 
 export TIMESTAMP=$(TZ=UTC /bin/date "+%Y%m%dT%H%M%SZ")
